xml/System.Windows.Media.Animation/IAnimation.xml

```diff
@@ -11,7 +11,7 @@
   </AssemblyInfo>
   <Interfaces />
   <Docs>
-    <summary>Interface to be implemented by an object that can animate a property value.</summary>
+    <summary>Provides support for animating a property value.</summary>
```

Comment:
Since this is an interface, it doesn't provide any support by itself.  It may be better to leave the original description here.

Reply:
It should follow our writing style of starting with a verb though @ryalanms 

https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Class

Reply:
Yes, I need helping getting the documentation congruent to the guidelines.  Thanks.

Reply:
That's what we're here for!